packages/react-vapor/src/components/sideNavigation/SideNavigationHeader.tsx

```diff
@@ -1,37 +1,40 @@
 import * as classNames from 'classnames';
 import * as React from 'react';
+
 import {Svg} from '../svg/Svg';
 
+/**
+ * @deprecated Use SideNavigationHeaderProps instead
+ */
 export interface ISideNavigationHeaderProps {
-    title: string;
+    title?: string;
     svgName?: string;
     svgClass?: string;
-    onClick?: () => void;
+    onClick?: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
 }
 
-export class SideNavigationHeader extends React.Component<ISideNavigationHeaderProps> {
-    private handleClick() {
-        if (this.props.onClick) {
-            this.props.onClick();
-        }
-    }
+/**
+ * @deprecated Will be removed in version 5
+ */
+export const SideNavigationHeader: React.FunctionComponent<ISideNavigationHeaderProps> = ({
+    onClick,
+    svgClass,
+    svgName,
+    children,
+}) => {
+    const icon = svgName ? (
+        <Svg
+            svgName={svgName}
+            svgClass={classNames('navigation-menu-section-header-icon icon mod-lg transparency-3', svgClass)}
+        />
+    ) : (
+        <span className="navigation-menu-section-header-icon" />
+    );
 
-    render() {
-        const svgClass = classNames(
-            'navigation-menu-section-header-icon icon mod-lg transparency-3',
-            this.props.svgClass
-        );
-        const icon = this.props.svgName ? (
-            <Svg svgName={this.props.svgName} svgClass={svgClass} />
-        ) : (
-            <span className="navigation-menu-section-header-icon" />
-        );
-        return (
-            <div className="navigation-menu-section-header text-white" onClick={() => this.handleClick()}>
-                {icon}
-                {this.props.title}
-                {this.props.children}
-            </div>
-        );
-    }
-}
+    return (
+        <div className="navigation-menu-section-header text-white" onClick={onClick}>
+            {icon}

```

Comment:
is `{this props title}` replaced by something else?

Reply:
Good catch I removed it by error.

Reply:
I reverted the changes to this whole file except the deprecation warning because I'm not using it anyway.